packages/components/src/components/hds/app-footer/link.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+
+import type { HdsInteractiveSignature } from '../interactive/';
+import type { HdsLinkColors, HdsLinkIconPositions } from '../link/types.ts';
+
+export interface HdsAppFooterLinkSignature {
+  Args: HdsInteractiveSignature['Args'] & {
+    color?: HdsLinkColors;
+    href?: string;
+    icon?: string;
+    iconPosition?: HdsLinkIconPositions;
+    isHrefExternal?: boolean;
+    isRouteExternal?: boolean;
+  };
+  Blocks: {
+    default: [];
+  };
+  Element: HdsInteractiveSignature['Element'];
```

Comment:
I don't think this is correct. The root element is an `Hds::AppFooter::Ite` so this should be:
```suggestion
  Element: HdsAppFooterItemSignature['Element'];
```